src/context/User.Context.js

```diff
@@ -201,6 +319,23 @@ const UserProvider = ({children}) => {
           isEditable,
           setIsEditable,
           editSet,
+          clearWorkout,
+          loading,
+          favouriteWorkoutData,
+          setFavouriteWorkoutData,
+          favouriteExerciseData,
+          setFavouriteExerciseData,
+          addFavouriteSet,
+          favouriteClickComplete,
+          favouriteDeleteSet,
+          favouriteEditSet,
+          getFavouriteTokens,
+          favouriteTokens,
+          title,
+          setTitle,
+          workoutNames,
+          setWorkoutNames,
+          getWorkoutNames,
```

Comment:
I think your user context is getting really large. It's a good idea to separate it out like FavouriteContext

Reply:
Noted for future projects, should be very little to no additional state or functions needed for this project